src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/WidgetsStyles.kt

```diff
@@ -148,22 +148,22 @@ class WidgetsStyles : Stylesheet() {
         }
 
         wizardCard {
-            maxHeight = 364.0.px
-            prefHeight = 364.0.px
-            prefWidth  = 364.0.px
+            maxHeight = 320.0.px
+            prefHeight = 320.0.px
+            prefWidth  = 300.0.px
             backgroundRadius += box(12.0.px)
-            backgroundColor += c("#CC4141")
-            textFill = c("#FFFF")
+            backgroundColor += c("#FFFF")
+            textFill = c("#CC4141")
             fontSize = 24.px
             effect = DropShadow(10.0, Color.GRAY)
-            cursor = Cursor.HAND
         }
 
         wizardCardButton {
```

Comment:
You might be able to move this inside the wizardCard rule to cut down on the custom rules
```kotlin
wizardCard {
    /* styling for card */
    button {
        /* styling for any button inside something with wizardCard class */
    }
}
```

Reply:
Just not sure if that plays well with JFXButton